gensim/models/wrappers/fasttext.py

```diff
@@ -220,6 +220,10 @@ def save(self, *args, **kwargs):
         super(FastText, self).save(*args, **kwargs)
 
     @classmethod
+    def load_word2vec_format(cls, *args, **kwargs):
+        return FastTextKeyedVectors.load_word2vec_format(*args, **kwargs)
```

Comment:
This was a bug in the flake8 script, fixed in cebb3fc16c5d588879bc2c7b0b8f84a3deed64e4